Talks/Brokers/Storages/IStorageBroker.Talks.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Talks.Models.Foundations.Talks;
+
+namespace Talks.Brokers.Storages
+{
+    public partial interface IStorageBroker
+    {
+        ValueTask<Talk> InsertTalkAsync(Talk talk);
+        IQueryable<Talk> SelectAllTalks();
```

Comment:
Method `SelectAllTalks` violates async broker conventions: return type should be `ValueTask` or `ValueTask<T>`; method name should end with `Async`.